src/js/drawer.js

```diff
@@ -105,6 +114,7 @@ function addDrawerToggles (drawerEl) {
 					drawerEl.focus();
 				}
 			});
+			drawerEl.addEventListener('keydown', handleTab);
```

Comment:
Similar to above, looks like `handleClose.addEvents` is added in a timeout to account for the transition. Does adding this event listener belong in that method?

Reply:
Ah, yeah... When starting this I misunderstood how all the `handleClose` stuff worked, but I think you're right!